src/JsonApi/Tests/Serializer/ItemNormalizerTest.php

```diff
@@ -153,21 +155,13 @@ public function testNormalizeCircularReference(): void
 
         $normalizer->setSerializer($this->prophesize(SerializerInterface::class)->reveal());
 
-        $circularReferenceLimit = 2;
-        if (!interface_exists(AdvancedNameConverterInterface::class) && method_exists($normalizer, 'setCircularReferenceLimit')) {
-            $normalizer->setCircularReferenceLimit($circularReferenceLimit);
-
-            $context = [
-                'circular_reference_limit' => [spl_object_hash($circularReferenceEntity) => 2],
-                'cache_error' => function (): void {},
-            ];
-        } else {
-            $context = [
-                'circular_reference_limit' => $circularReferenceLimit,
-                'circular_reference_limit_counters' => [spl_object_hash($circularReferenceEntity) => 2],
-                'cache_error' => function (): void {},
-            ];
-        }
+        // Symfony >= 7.3
+        $splObject = class_exists(EventStreamResponse::class) ? spl_object_id($circularReferenceEntity) : spl_object_hash($circularReferenceEntity);
```

Comment:
We need this for fixing JsonApi test since this change in Symfony 7.3 : https://github.com/symfony/symfony/pull/59355